tests/components/RangeInput.stories.tsx

```diff
@@ -18,9 +18,12 @@ const meta: Meta<typeof RangeInput> = {
   },
   decorators: [(Story) => (
     <SearchHeadlessContext.Provider value={generateMockedHeadless()}>
-      <FilterGroupContext.Provider value={filterGroupContextValue}>
+      <FilterGroupProvider
+          fieldId={'123'}
+          defaultExpanded={true}
```

Comment:
I think defaultExpanded is optional. can we remove this prop unless we need the behavior to be different from the default?